src/borg/threading/create.py

```diff
@@ -624,6 +624,13 @@ def pack_and_add_item(self):
         self.num_items += 1
         if 'chunks' in item:
             self.archive.stats.nfiles += 1
+        try:
+            # XXX This is the sledgehammer approach of getting rid of these, see comment in Item
+            del item.original_path
+            del item.status
+            del item.num_chunks
+        except AttributeError:
+            pass
```

Comment:
hmm, you seem to expect that any of them could be missing (thus the scope around all 3).
but the first one missing will raise and the remaining ones won't be killed.
so, 3x try/except?